tt_eager/tt_dnn/op_library/untilize/untilize_with_halo_op_v2.cpp

```diff
@@ -230,16 +230,10 @@ operation::ProgramWithCallbacks untilize_with_halo_multi_core_v2(
             const std::vector<std::optional<const Tensor>>&,
             const std::vector<Tensor>& output_tensors) {
             auto src_buffer = input_tensors.at(0).buffer();
-            auto padding_config_buffer = input_tensors.at(1).buffer();
-            auto local_config_buffer = input_tensors.at(2).buffer();
-            auto remote_config_buffer = input_tensors.at(3).buffer();
             auto dst_buffer = output_tensors.at(0).buffer();
 
             UpdateDynamicCircularBufferAddress(program, src_cb, *src_buffer);
             UpdateDynamicCircularBufferAddress(program, out_cb, *dst_buffer);
-            UpdateDynamicCircularBufferAddress(program, padding_config_cb, *padding_config_buffer);
-            UpdateDynamicCircularBufferAddress(program, local_config_cb, *local_config_buffer);
-            UpdateDynamicCircularBufferAddress(program, remote_config_cb, *remote_config_buffer);
```

Comment:
Why are these not needed anymore?

Reply:
Config buffers are part of program now. The addresses won't be changed so there is no need to update.

Reply:
Hmm, maybe there should be something to indicate this, instead of relying on guarantees outside of the op.

Reply:
Making it part of program object guarantees its persistence in memory until program is destructed

Reply:
Also, config tensors are no longer part of the vector of input tensors. I deleted the code in this change which was incorrectly trying to access config tensors from input tensors vector.

Reply:
It looks like it's still passed in as input tensor? 
<img width="1356" alt="image" src="https://github.com/tenstorrent/tt-metal/assets/109366241/9b7c4e0d-a4aa-47ae-8ccd-8e62ce8a6e14">

Reply:
This is the old halo op which will be deprecated. The refactored halo is in halo_op.cpp file.